admin-dev/themes/new-theme/js/pages/product/edit/manager/feature-values-manager.ts

```diff
@@ -120,7 +120,7 @@ export default class FeatureValuesManager {
   private watchFeatureSelectors(): void {
     $(this.$collectionContainer).on('change', ProductMap.featureValues.featureSelect, (event) => {
       const $selector = $(event.target);
-      const idFeature = Number($selector.val());
+      const featureId = Number($selector.val());
```

Comment:
I am not sure this is related to the PR goal 😛 "Clean old product page code"

Reply:
It is because I had to move an "API" that was in the old feature controller into the new one, and the placeholder was not good so I had to rename it and adapt the related code